runtime/runtime-params-estimator/src/testbed_runners.rs

```diff
@@ -109,18 +110,15 @@ pub fn measure_actions(
 }
 
 // TODO: super-ugly, can achieve the same via higher-level wrappers over POSIX read().
-#[cfg(any(target_arch = "x86_64"))]
+#[cfg(target_family = "unix")]
 #[inline(always)]
-pub unsafe fn syscall3(mut n: usize, a1: usize, a2: usize, a3: usize) -> usize {
-    llvm_asm!("syscall"
-         : "+{rax}"(n)
-         : "{rdi}"(a1) "{rsi}"(a2) "{rdx}"(a3)
-         : "rcx", "r11", "memory"
-         : "volatile");
-    n
+pub unsafe fn syscall3(fd: u32, buf: &mut [u8]) {
```

Comment:
Think we can do that simpler, like is being done in https://github.com/near/nearcore/pull/3705/files#diff-476f808c2693b5c35256c89880c018915744db6a28359ef8132fcb6d5184b2ac

Reply:
That looks good.  Since you are updating the API in that PR, is it okay to just merge this as is since it works and then update  with your PR?

Reply:
NVM I updated this PR.